course_metadata/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf import settings
+from django.conf.urls import patterns, url
+
+from . import views
+
+COURSE_ID_PATTERN = settings.COURSE_ID_PATTERN
+
+urlpatterns = patterns(
+    '',
+    url(r'^{}/settings$'.format(COURSE_ID_PATTERN), views.CourseSettingView.as_view(), name='course_settings'),
```

Comment:
can we make view name `course_settings` bit unique. may be `additional_course_settings`?